spec/lib/migrations/load_rating_factors_spec.rb

```diff
@@ -116,7 +116,7 @@
     private
 
     def invoke_task
-      Rake::Task["load_rating_factors:update_factor_sets"].execute({:file_name => "#{Rails.root}/spec/test_data/plan_data/rate_factors/2017/SHOP_RateFactors_CY2017_SOFT_DRAFT.xlsx"})
```

Comment:
This loading task needs  be less brittle

Reply:
When ever we load the plans or update the plans we should run this rake task.